### PR TITLE
Add 1.5.0 draft release notes

### PR DESCRIPTION
**Pull request by spiffxp**

### CODE REVIEW

**Comment by spiffxp** [comment]
I signed it!

**Comment by sarahnovotny** [comment]
LGTM

File: release-1.5/release-notes-draft.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+## Action Required Before Upgrading
+
+* batch/v2alpha1.ScheduledJob has been renamed, use batch/v2alpha1.CronJob instead ([#36021](https://github.com/kubernetes/kubernetes/pull/36021), [@soltysh](https://github.com/soltysh))
+* PetSet has been renamed, use StatefulSet instead ([docs](https://deploy-preview-1704--kubernetes-io-vnext-staging.netlify.com/docs/tasks/manage-stateful-set/upgrade-pet-set-to-stateful-set/)) ([#35663](https://github.com/kubernetes/kubernetes/pull/35663), [@janetkuo](https://github.com/janetkuo))
+* If you are upgrading your Cluster Federation components from v1.4.x, please update your `federation-apiserver` and `federation-controller-manager` manifests to the new version ([#30601](https://github.com/kubernetes/kubernetes/pull/30601), [@madhusudancs](https://github.com/madhusudancs))
+* The deprecated kubelet --configure-cbr0 flag has been removed, and with that the "classic" networking mode as well.  If you depend on this mode, please investigate whether `kubenet` can meet your needs. ([#34906](https://github.com/kubernetes/kubernetes/pull/34906), [@luxas](https://github.com/luxas))
```
Comment (luxas) [inline release-1.5/release-notes-draft.md]: whether the other network plugins `kubenet` or `cni` meet your needs

File: release-1.5/release-notes-draft.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- **Auth**
+  - [alpha] Further polishing of the Role-based access control alpha API including a default set of cluster roles. ([docs](http://kubernetes.io/docs/admin/authorization/)) ([kubernetes/features#2](https://github.com/kubernetes/features/issues/2))
+  - [beta] Added ability to authenticate/authorize access to the Kubelet API ([docs](https://deploy-preview-1684--kubernetes-io-vnext-staging.netlify.com/docs/admin/kubelet-authentication-authorization/)) ([kubernetes/features#89](https://github.com/kubernetes/features/issues/89))
+- **AWS**
+  - [stable] Roles should appear in kubectl get nodes ([kubernetes/features#113](https://github.com/kubernetes/features/issues/113))
+- **Cluster Lifecycle**
+  - [alpha] Use kubeadm to make it easy to get a cluster running. ([docs](https://deploy-preview-1684--kubernetes-io-vnext-staging.netlify.com/docs/admin/kubeadm/)) ([kubernetes/features#11](https://github.com/kubernetes/features/issues/11))
```
Comment (luxas) [inline release-1.5/release-notes-draft.md]: Maybe: [alpha] Improved UX and usability for the `kubeadm` binary that makes it easy to get a new cluster running

The link should be `docs/getting-started-guides/kubeadm/`

File: release-1.5/release-notes-draft.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+[Documentation](http://kubernetes.github.io) & [Examples](http://releases.k8s.io/release-1.5/examples)
+
+## Major Themes
+
+- Improved Federation Support
+  - Daemonsets
+  - Deployments
+  - ConfigMaps
+- Simplified Cluster Deployment
+  - `kubeadm`
```
Comment (luxas) [inline release-1.5/release-notes-draft.md]: maybe: `kubeadm` improvements

File: release-1.5/release-notes-draft.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  - `kubefed`
```
Comment (luxas) [inline release-1.5/release-notes-draft.md]: maybe: `kubefed`: new binary

or something that makes it clear that kubefed is a new binary and kubeadm has just been improved

File: release-1.5/release-notes-draft.md (hunk 3 above)
Comment (idvoretskyi) [inline release-1.5/release-notes-draft.md]: +1. We already have `kubeadm` available, here we are focusing on improvements stuff more.

**Pull request merged**